load/load_distributor.go

```diff
@@ -1,12 +1,44 @@
 package load
 
-import "fmt"
+import (
+	"fmt"
+
+	consumer "github.com/AdityaMayukhSom/ruskin/consumer"
+)
+
+// Making a map of TopicId and LoadPartition
+// The map values have to be mapped to LoadPartition uid(will implement later)
+var TopicToPartitionMap = make(map[string]*LoadPartition)
 
 type LoadDistributor struct {
 	PartitionsCount int
 	Partitions      []LoadPartition
 }
 
+func (ld *LoadDistributor) getOrCreatePartition(topic string) *LoadPartition {
```

Comment:
If this function is supposed to be used outside this package, the name should start with capital letter, also GetPartition name with sufficient documentation seems like a better name than getOrCreatePartition as the name does not specify what it does after creation of the partition.